.qubesbuilder

```diff
@@ -21,3 +21,4 @@ vm:
       - "sed -i 's/$(PA_VER)/1:$(PA_VER)/g' @SOURCE_DIR@/debian/rules"
       - 'sed -i /Trolltech.conf/d @SOURCE_DIR@/debian/qubes-gui-agent.install'
       - '@PLUGINS_DIR@/source_deb/scripts/debian-quilt @SOURCE_DIR@/series-debian-vm.conf @SOURCE_DIR@/debian/patches'
+      - 'sed -i "s/pulseaudio-qubes,/pipewire-qubes,/" @SOURCE_DIR@/debian/control'
```

Comment:
In fact, this is no-op (not matching anything)...

Reply:
yes because it's the wrong repo...

Reply:
https://github.com/QubesOS/qubes-meta-packages/pull/59